src/Symfony/Component/AssetMapper/ImportMap/RemotePackageStorage.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\ImportMap;
+
+/**
+ * Manages the local storage of remote/vendor importmap packages.
+ */
+class RemotePackageStorage
+{
+    public function __construct(
+        private readonly string $vendorDir,
+    ) {
+    }
+
+    public function getStorageDir(): string
+    {
+        return $this->vendorDir;
+    }
+
+    public function isDownloaded(ImportMapEntry $entry): bool
```

Comment:
This should add proper error handling for the case of passing a non-remote entry to this method (as this is not an internal API where we can rely on the caller doing the right thing).
right now, we would get an error about `$entry->packageModuleSpecifier` being uninitialized.